beacon-chain/p2p/config.go

```diff
@@ -6,6 +6,9 @@ import (
 	"github.com/prysmaticlabs/prysm/v4/beacon-chain/startup"
 )
 
+// This is the default queue size used if we have specified an invalid one.
+const defaultPubsubQueueSize = 32
```

Comment:
Something smells funny here.
Flag has its default at 600
This file has its default at 32

Scenarios
- if user specifies nothing, then 600 is used
- If user species a >0 value `N`, then `N` is used
- if user specifies 0, then 32 is used

Should this be 600?

Reply:
32 was the default set by gossipsub, but if our default is 600 then we should probably just use that. Changing it